src/hooks/useResponsiveLayout/index.native.ts

```diff
@@ -32,7 +23,15 @@ type ResponsiveLayoutResult = {
  * For more details on the various modal types we've defined for this app and implemented using react-native-modal, see `ModalType`.
  */
 export default function useResponsiveLayout(): ResponsiveLayoutResult {
-    const {isSmallScreenWidth, isExtraSmallScreenHeight, isExtraSmallScreenWidth, isMediumScreenWidth, isLargeScreenWidth, isSmallScreen} = useWindowDimensions();
+    const {windowWidth, windowHeight} = useWindowDimensions();
+
+    const isExtraSmallScreenHeight = windowHeight <= variables.extraSmallMobileResponsiveHeightBreakpoint;
+    const isSmallScreenWidth = true;
+    const isMediumScreenWidth = false;
+    const isMediumOrLargerScreenWidth = false;
```

Comment:
```suggestion
    const isMediumOrLargerScreenWidth = windowWidth > variables.mobileResponsiveWidthBreakpoint;
```

```
/**
 * The main difference between useOnboardingLayout and useWindowDimension is that
 * useWindowDimension hardcodes isSmallScreenWidth, isMediumScreenWidth and
 * isLargeScreenWidth on native platforms, while this hook below always takes
 * screen width into consideration, no matter the platform.
 */
```

^ a comment from `useOnboardingLayout`. I think we should assign the same value here as well, instead of hardcoding it.